people/titles/engineer.md

```diff
@@ -0,0 +1,33 @@
+# Open Source Infrastructure Engineer
+
+An Open Source Infrastructure (OSIE) focuses on infrastructure that supports interactive computing.
+It intersects job titles such as “dev-ops engineer”, “site reliability engineer”, “software engineer”, and “cloud engineer”.
+
+## Key responsibilities
```

Comment:
All the content on this page resembles a lot of the job posts we publish which, I think, makes sense.

Reply:
That is one of the goals - we should be able to re-use the content here relatively quickly for job posts.